scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.12.x_to_v4.0.x.txt

```diff
@@ -2,7 +2,9 @@ Steps to migrate from v3.12.x to v4.0.x
 
 1. Copy the file : migration_v3.12.x_to_v4.0.x.py file from: <script source path>
 2. Run the below command to execute the script.
-
+3. migration added true/false 
```

Comment:
two minor change here - 
1. change this text "migration added true/false" -> "set migration-flag to" 
2. move this to the bottom of file (after example)

Reply:
Updated as suggesstion